app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,32 @@
+class LikesController < ApplicationController
+  def index
+    @likes = Likes.all
+  end
+  
+  def create
+    @bookmark = Bookmark.find(params[:bookmark_id])
+    like = current_user.likes.build(bookmark: @bookmark)
+    
+    if like.save
+      flash[:notice] = "Bookmark liked!"
+      redirect_to(@bookmark.topic)
+    else
+      flash[:alert] = "Error!"
+      redirect_to(@bookmark.topic)
+    end
+  end
+  
+  def destroy
+    @bookmark = Bookmark.find(params[:bookmark_id])
+    
+    like = current_user.likes.find(params[:id])
+    
+    if like.destroy
```

Comment:
@ianroberts131 How could the repetition of lines 26 and lines 29 in the `if/else` branches be avoided?